packages/editor/src/components/autosave-monitor/test/index.js

```diff
@@ -8,7 +8,9 @@ import { render } from '@testing-library/react';
  */
 import { AutosaveMonitor } from '../';
 
-jest.useFakeTimers( { legacyFakeTimers: true } );
+jest.useFakeTimers();
+jest.spyOn( global, 'clearTimeout' );
```

Comment:
However, it was necessary to start spying on `clearTimeout` and `setTimeout`, as also displayed on the [jest docs page](https://jestjs.io/docs/timer-mocks).

It's odd that it worked before. I assume some cross-test-setup bleeding was going on.